### ISSUE TITLE
AttributeError: 'Key' object has no attribute 'startswith'

### ISSUE THREAD

**Issue by sirkitree** [socra/socra-python#4]
I thought I'd try to use socra to improve the readme and address #3 but having troubel running it.

I installed with `pip install socra`.
It seems to run and I get a version and help.

Here is a command I gave:
`socra improve README.md "add instructions for what api keys the program looks for and explain how to add them"`

Here is the output:
```
Traceback (most recent call last):
  File "/opt/homebrew/bin/socra", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/socra/cli.py", line 34, in improve
    output = a.run(a.Inputs(node=node, prompt=prompt))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/socra/nodes/actions/root.py", line 111, in run
    resp = cr.process()
           ^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/socra/completions/base.py", line 74, in process
    llm = ChatOpenAI(model=self.model.key)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/langchain_core/load/serializable.py", line 111, in __init__
    super().__init__(*args, **kwargs)
  File "/opt/homebrew/lib/python3.11/site-packages/pydantic/main.py", line 212, in __init__
    validated_self = self.__pydantic_validator__.validate_python(data, self_instance=self)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/langchain_openai/chat_models/base.py", line 501, in validate_temperature
    if model.startswith("o1") and "temperature" not in values:
       ^^^^^^^^^^^^^^^^
AttributeError: 'Key' object has no attribute 'startswith'
```

**Comment by morteymike** [comment]
Hello, thank you for the interest and the first issues!!
As you can see, the library is under heavy development, so expect to see some big changes in the coming days to how the tool looks, how the docs read, and so on.

As for this issue, I've just released `socra==0.0.7` to address this issue (hopefully).

**Comment by sirkitree** [comment]
Yup, totally fixed now. I ran `pip install --upgrade socra` and the new version works without error, at least for this command :)
